spec/requests/api/v1/job_applications/job_application_create_spec.rb

```diff
@@ -44,6 +44,20 @@
         expect(jobApp[:data][:attributes][:contact_information]).to eq(job_application_params[:contact_information])
         expect(jobApp[:data][:attributes][:company_id]).to eq(job_application_params[:company_id])      
       end
+      it "Allows 2 unique users to create job applications with the same URL" do
```

Comment:
This is an excellent test to include, not only does it test the flexibility of the validation, but this functionality is necessary for the foundation of this application.